kuma/static/js/wiki-edit-draft.js

```diff
@@ -0,0 +1,428 @@
+        if (prevDraft && draftMatchesStart && !isErrors) {
+            // don't need draft if it's a copy of the starting content
+            clearDraft(false);
+        }
+
+        // if draft does not match current content, or if there are errors
+        if (prevDraft && !draftMatchesStart) {
+            var draftRevision = localStorage.getItem(DRAFT_NAME + '#revision');
+            var draftTime = localStorage.getItem(DRAFT_NAME + '#save-time');
+            var pageRevision = $revisionInputs.val();
+            var draftTimeStr = draftTime ? draftTime : gettext('an unknown date');
+            var draft = gettext('You have a draft from:');
+            var restore = gettext('Restore draft.');
+            var discard = gettext('Discard draft.');
+            var draftWarning = '';
+            var draftType;
+            if(draftRevision === pageRevision || isNewTranslation) {
+                // draft was created on this revision or is new translation
+                draftType = 'current';
+            } else if (draftRevision === '' && pageRevision !== '') {
+                // draft of translation that has since been published
+                draftType = 'old';
+            } else if (draftRevision !== null && draftRevision < pageRevision) {
+                // revision string is out of date
+                draftType = 'old';
+            } else {
+                // created before we were saving revision numbers
+                // or something went wrong
+                draftType = 'unknown';
+            }
+
+            if (draftType === 'old') {
+                restore = gettext('View draft.');
+                $draftOld.addClass('warning');
+                draftWarning += '<div class="readable-line-length">';
+                // long line is long, but breaking it up wrecks it for the localizers
+                draftWarning += gettext('A newer version of this article has been published since this draft was saved. You can restore the draft to view the content, but you will not be able to submit it for publishing.');
+                draftWarning += ' <a href="#TODO" class="external-icon">Revision history.</a> ';
+                draftWarning += '<a href="#TODO" class="external-icon">Published version</a>';
+                draftWarning += '</div>';
+
+                mdn.analytics.trackEvent({
+                    category: 'draft',
+                    action: 'outdated-draft-warning'
+                });
+            }
+
+            if (draftType === 'unknown') {
+                $draftOld.addClass('note');
+                draftWarning += '<div class="readable-line-length">';
+                // long line is long, but breaking it up wrecks it for the localizers
+                draftWarning += gettext('Compare this date to the latest revision date to ensure you\'re not overwriting later changes.');
+                draftWarning += ' <a href="#TODO" class="external-icon">Revision history.</a>';
```

Comment:
``Revision history.`` should be localized too.  Which will make this line even longer...